src/preset_cli/cli/superset/sync/dbt/databases.py

```diff
@@ -78,6 +80,16 @@ def sync_database(  # pylint: disable=too-many-locals, too-many-arguments
             **meta,
         )
 
-    database["sqlalchemy_uri"] = connection_params["sqlalchemy_uri"]
```

Comment:
Previously, this would always be executed (which pretty much builds the `SQLAlchemy` URI from the connection settings on the `profiles.yml`). Now it's only executed in case `--import-db` is passed.